applications/nrf5340_audio/README.rst

```diff
@@ -86,10 +86,14 @@ Connected Isochronous Stream (CIS)
   This is the default configuration of the nRF5340 Audio application.
   In this configuration, you can use the nRF5340 Audio development kit in the role of the gateway, the left headset, or the right headset.
 
+  In the current version of the nRF5340 Audio application, the CIS mode offers both unidirectional and bidirectional communication.
+  You can configure the CIS to bidirectional communication, in which it will support a walkie-talkie demonstration.
```

Comment:
"to" -> "for"

Reply:
Shall "communication" be used? The CIS always has bidir communication. Suggest bidirectional audio streams